.pre-commit-config.yaml

```diff
@@ -63,7 +63,7 @@ repos:
         args:
             - --license-filepath
             - license_header.txt
--   repo: https://gitlab.com/pycqa/flake8
+-   repo: https://gitlab.com/pycqa/flake8.git
```

Comment:
Not sure what this is 🙃

Reply:
it is just to fix the git issue on my dev vm
```
error: RPC failed; result=22, HTTP code = 404
fatal: The remote end hung up unexpectedly
```